exchanges/btse/btse_test.go

```diff
@@ -752,6 +752,30 @@ func TestStripExponent(t *testing.T) {
 	assert.ErrorIs(t, err, errInvalidPairSymbol, "Should error on a symbol with too many underscores")
 }
 
+func TestMarketPair(t *testing.T) {
+	t.Parallel()
+
+	for _, test := range []struct {
+		symbol         string
+		base           string
+		quote          string
+		futures        bool
+		expectedErr    error
+		expectedSymbol string
+	}{
+		{symbol: "RUNEPFC", base: currency.RUNE.String(), quote: currency.USD.String(), futures: true, expectedSymbol: "RUNEPFC"},
+		{symbol: "TRUMPPFC", base: "TRUMPSOL", quote: currency.USD.String(), futures: true, expectedSymbol: "TRUMPPFC"},
+		{symbol: "BTCUSD", base: "NAUGHTYBASE", quote: currency.USD.String(), futures: true, expectedErr: errInvalidPairSymbol},
+		{symbol: "NAUGHTYSYMBOL", base: currency.BTC.String(), quote: currency.USD.String(), expectedErr: errInvalidPairSymbol},
+		{symbol: "BTC-USD", base: currency.BTC.String(), quote: currency.USD.String(), expectedSymbol: currency.NewPair(currency.BTC, currency.USD).String()},
```

Comment:
Nit: What we expect shouldn't depend on gct consts at all, because it's btse related. So a hardcoded expected symbol is a better test here. I know it's the same thing, except it's not 😁 
```suggestion
		{symbol: "BTC-USD", base: currency.BTC.String(), quote: currency.USD.String(), expectedSymbol: "BTCUSD"},
```